### PR TITLE
Assorted ES Mapping changes

### PR DESCRIPTION
**Pull request by sravfeyn**
I found these changes as I was working on ES7 changes.

These fields don't need to be analyzed as the underlying data is always a keyword. I am not updating the mapping as a reindex is not necessary.

@millerdev @snopoke @calellowitz

### CODE REVIEW

File: corehq/pillows/mappings/case_mapping.py [hunk 1]
```diff
@@ -41,7 +41,8 @@
                     },
                     'type': 'nested'},
         'closed': {'type': 'boolean'},
-        'closed_by': {'type': 'string'},
+        # todo; check with team
```
Comment (millerdev) [inline corehq/pillows/mappings/case_mapping.py:44]: Do these need to be addressed before the PR is merged?

**Review by snopoke — APPROVED** [review]
Pending question on "#todo" comments

File: corehq/pillows/mappings/case_mapping.py (hunk 1 above)
Comment (sravfeyn) [reply corehq/pillows/mappings/case_mapping.py:44]: >  I am not updating the mapping as a reindex is not necessary.

This is what I wanted to check is okay. I guess that's okay, I will remove the todo comments.

Comment (sravfeyn) [reply corehq/pillows/mappings/case_mapping.py]: This is good to go?

Comment (millerdev) [reply corehq/pillows/mappings/case_mapping.py]: It is still not clear if you verified that the comments needed to be addressed or not. :+1: from me if they are no longer important.

**Review by snopoke — APPROVED** [review]

**Pull request merged**